### ISSUE TITLE
Scrolling on scrollbar does not trigger pointerup in IE10+11

### ISSUE THREAD

**Issue by AllNamesRTaken** [hammerjs/hammer.js#478]
I believe this is an IE bug!

IE does not trigger pointerup/cancel etc when releasing the scrollbar. Though it for some reason triggers pointermove once when realeasing.

This causes Hammer to remember the touchstart without ever becoming the touchend. Consequently preventing the next tap within the container to trigger tap.

This happens in IE and when for example a tap handler is bound to the body to minimize the number of events in large applications.

As you might expect clicking anywhere within the container triggers the touchend and makes the next tap event register.

I filed  a bug and created a replication here (IE11 only or you have to change to MsPointer events): http://jsfiddle.net/u4NtF/3/

**Comment by ziaulmurtaza** [comment]
Hi tulach,

I was facing the same issue, and I pleased to see this post. I download hammer.js version 1.0.10 update bindDomOnTouch function with your solution add this updated hammer.js file in my project and bind my scroll and mouseup event with hammer.js. It worked. Thanks alot for your worthy post. :+1:

**Comment by runspired** [comment]
@arschmitz this looks like an issue that simply needs turned into a PR

**Comment by tulach** [comment]
@ziaulmurtaza If you use touch and release events instead of mousedown/up, it should work (I validated that)

mc.on('touch', function() { mousestate="Down";console.log('down'); });
mc.on('release', function() { mousestate="Up";console.log('up'); });

**Comment by tulach** [comment]
See PR #810 . It violates declared rules (not to patch compiled files) but hope this help a bit.

**Comment by ziaulmurtaza** [comment]
@tulach,

I updated the hammer.js and updated mouse down and mouse up events to touch and release. release event is now triggering but not according to mouse release. it is triggering right after mouse down even I didn't release mouse. so not resolving my problem. can you please look into it. 

I will be very thankful to you.

**Comment by tulach** [comment]
Sorry, I cannot replicate that :(. I've tested IE9,10,11.
Go to URL http://controlsjs.com/demos/HammerIE/ , open console (F12) and try to click. In console output you will see "down" and "up" texts which should be displayed according to your mouse press and release.

**Comment by ziaulmurtaza** [comment]
@tulach I can help you reproducing this, 
open url http://controlsjs.com/demos/HammerIE/ press F12 (open developer tools) go to console tab. to view results. now start scrolling you will see down and up messages in console which means touch and released events have been called even you never release mouse. please make sure you are testing on IE.

**Comment by tulach** [comment]
![hammerie](https://cloud.githubusercontent.com/assets/3995083/8277711/cecd02c4-18c5-11e5-92e5-a7a9a90c73cd.png)
Done that. One mouse down and one mouse up is in my console log. Right according to my mouse press and release. Tested on Win7Pro 64bit IE11.

**Comment by ziaulmurtaza** [comment]
@tulach, 
let me check on win7 but I am on win8.1 and IE11 it is not working for me. when I start scrolling mouse down and up are triggering. mouse up should trigger when I finish scrolling(release mouse). it is working fine on Chrome but not in IE.

**Comment by tulach** [comment]
Yes, you are right. On Win8.1 IE11 mouse up occurs right after mouse down:(. But I see it is fired only once. It's not perfect but better than loosing mouse up entirely.
